aws/resource_aws_rds_cluster_test.go

```diff
@@ -2698,9 +2698,10 @@ resource "aws_db_subnet_group" "test" {
 }
 
 resource "aws_rds_cluster" "restored_pit" {
-  cluster_identifier  = "%s"
-  skip_final_snapshot = true
-  engine              = aws_rds_cluster.test.engine
+  cluster_identifier              = "%s"
+  skip_final_snapshot             = true
+  engine                          = aws_rds_cluster.test.engine
+  enabled_cloudwatch_logs_exports = ["audit", "error"]
```

Comment:
this is a great start! let's just move this specific configuration check (with enabled cloudwatch log exports) to a new test if you don't mind so we can confirm and isolate the new expected behavior as well as check the attribute is used and set appropriately